var/spack/repos/builtin/packages/tau/package.py

```diff
@@ -1,5 +1,5 @@
 ##############################################################################
-# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
```

Comment:
We should probably leave 2018 here...

Reply:
I have updated the TAU package after cleaning it up and have just pushed.